scripts/utils.js

```diff
@@ -62,3 +62,11 @@ export function decorateLinkedPictures(container) {
       a.setAttribute('title', txt);
     });
 }
+
+/**
+ * Checks whether device is desktop or not.
+ * 
+ */
+export function isDesktop() {
+  return window.innerWidth < 1280;
```

Comment:
this method should return true if screen size is desktop screen, but currently its returning false.